src/components/collabs/pick.svelte

```diff
@@ -27,11 +27,13 @@
 			return;
 		}
 
-		const response = _window.confirm($t('collabs.delete_pick_confirm'));
-
-		if (response) {
-			await axios.delete('/api/collabs/' + collab.id + '/picks/' + pick.id);
+		const reasonResponse = _window.prompt('Whats the reason for the deletion?', 'Duplicate Pick');
```

Comment:
Only ask for this if the user is not deleting their own pick.

Reply:
Removed it from asking for a reason instead I used a window.confirm instead of the user deleting their own pick.